### PR TITLE
 Manage quota only for containers created by Swarm #48

### PR DESCRIPTION
**Pull request by tsofnat**
on delete container it is checked that the id exists on quota's struct
on refresh loop check that cluster container has the swarm label added to headers: com.docker.swarm.id

### CODE REVIEW

File: pkg/multiTenancyPlugins/quota/quota.go [hunk 1]
```diff
@@ -80,11 +80,6 @@ func (quotaImpl *DefaultQuotaImpl) Handle(command string, cluster cluster.Cluste
 	case "containerdelete":
```
Comment (ezrasilvera) [inline pkg/multiTenancyPlugins/quota/quota.go]: Why did you remove the check? what happen if I delete through Swarm a container that created by K8 ? Are you going to decrease the quota ?

Comment (tsofnat) [inline pkg/multiTenancyPlugins/quota/quota.go]: the check is done in quotaMgmt.go , in function DecreaseQuota.
it checks if such container id exists on the tenants list (if contInfo, ok := tenantQuota.containers[id]; ok{ ),
only swarm containers are in this list so k8 container will not be and quota won't be decreased.
